daffodil-lib/src/test/scala/org/apache/daffodil/xml/test/unit/TestXMLUtils.scala

```diff
@@ -284,7 +284,7 @@ class TestXMLUtils {
   @Test def testBlobDiff01() {
     val path1 = createBlobFile("A1B2C3")
     val path2 = createBlobFile("A1B2C3")
-    val diff = XMLUtils.computeBlobDiff("path", path1.toString, path2.toString)
+    val diff = XMLUtils.computeBlobDiff("path", path1.toUri.toString, path2.toUri.toString)
```

Comment:
wasn't expecting these fixes in this PR. I'm ok with it, just isn't mentioned in the description.

Reply:
When I made the PR, I didn't realize there were actual failures (I noticed this originally in another PR that did have failures). But it turned out, we've had tests failing on windows for a while and I didn't notice until the option was removed and a build happened. Since it was kindof related, I quickly fix the tests so they'd pass on windows. I'll fix the commit message when I squash everything together once all the tests pass.